eox_tagging/validators.py

```diff
@@ -0,0 +1,68 @@
+""" File to define validations for tag model fields
+"""
+
+import re
+
+from django.conf import settings as base_settings
+from django.core.exceptions import ValidationError
+
+
+class Validators(object):
+    """ Defines all validator methods.
+    """
+
+    def __init__(self, instance):
+        """
+        Attributes:
+            instance: instance of the model to validate before saving
+        """
+        self.instance = instance
+        self.functions = {  # Functions defined for validation
+            "definition": self.__validate_field_definition
+        }
+
+    def run_validators(self):
+        """
+        Runs defined validators
+        """
+        self.__validate_not_update()
+
+        definitions = base_settings.EOX_TAGGING_DEFINITIONS
```

Comment:
What happens if the definition does not exist?

Reply:
Attribute error not handled. Added the setting to common ✓